mobile/babel.config.js

```diff
@@ -2,5 +2,7 @@ module.exports = function(api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: ['transform-inline-environment-variables'],
```

Comment:
I think this is only necessary for .env file support, so we can remove it with the .env file.

Reply:
We need it if we do not have the app.config file. The reason Aksharan and I did not want to use it is because in the EXPO documentation, it says that app.config.ts is experimental and can change at anytime, so we thought it was best to avoid it

Reply:
Referring to this? https://docs.expo.io/workflow/configuration/#using-typescript-for-configuration-appconfigts-instead-of

Let's just use app.config.js then, it's just build configuration, type safety isn't as big a deal

Reply:
Yup. No worries then, we can get that change done quickly. Will resolve all of the babel stuff too.